### DISCUSSION TITLE
AWS SM

### DISCUSSION THREAD

**Discussion by xenedium** [DaddaAdam/Spring-Reddit#28] [Q&A] [answered]
Is it necessary to use AWS Secrets Manager ?

**Comment by DaddaAdam** [comment, marked as answer]
Protecting credentials is absolutely necessary, but there are alternatives to AWS SM (Hashicorp Vault, Azure Key Vault, Google Cloud KMS...)
AWS Secrets Manager was just a personal preference

**Comment by imvikysharma** [comment]
It depends on your specific use case and security requirements. If you have sensitive information such as passwords, tokens, or keys that need to be stored securely, then AWS Secrets Manager can be a good option.
And I would like to recommend 1 book on AWS. I am using it personally. Do have a look.

I would like to recommend a book on AWS

https://www.amazon.com/dp/B0BVJKH8PY?maas=maas_adg_AA06FAB7196A55FE3DBBEE7958CED70A_afap_abs&ref_=aa_maas&tag=maas
